.github/workflows/release.yml

```diff
@@ -0,0 +1,21 @@
+name: Deployment
+on:
+    release:
+        types: [published]
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@v2
+            -   name: Ballerina Build
+                uses: ballerina-platform/ballerina-action/@master
+                with:
+                    args:
+                        build -a -c --skip-tests
```

Comment:
As we discussed earlier, shall we enable tests for the deployment workflow. The connection settings should be provided as environment variables through GitHub secrets.

Reply:
Fixed in 3434e87f2eaebb26f39cac1e10ecd71b1e7520a4.